pkg/component/logging/fluentoperator/custom_resources.go

```diff
@@ -18,168 +18,66 @@ import (
 	"context"
 
 	fluentbitv1alpha2 "github.com/fluent/fluent-operator/v2/apis/fluentbit/v1alpha2"
-	corev1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	v1beta1constants "github.com/gardener/gardener/pkg/apis/core/v1beta1/constants"
 	"github.com/gardener/gardener/pkg/client/kubernetes"
 	"github.com/gardener/gardener/pkg/component"
-	"github.com/gardener/gardener/pkg/component/logging/fluentoperator/customresources"
-	kubernetesutils "github.com/gardener/gardener/pkg/utils/kubernetes"
 	"github.com/gardener/gardener/pkg/utils/managedresources"
 )
 
 const (
 	// CustomResourcesManagedResourceName is the name of the managed resource which deploys the custom resources of the operator.
-	CustomResourcesManagedResourceName = OperatorManagedResourceName + "-custom-resources"
+	CustomResourcesManagedResourceName = "flb-custom-resources"
```

Comment:
Since the ManagedResource is already deployed today, the name cannot simply changed. Our proposal is to stick to `fluent-operator-custom-resources` in the seed case and use `fluent-operator-custom-resources-garden` for the garden cluster.